marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/CallBatcherImpl.java

```diff
@@ -471,8 +492,22 @@ private void initialize() {
         started.set(true);
         
         if(callArgsGenerator != null) {
+            Forest[] forests;
+            String[] forestNames = null;
+            if(forestParamName!=null) {
+                forests = super.getForestConfig().listForests();
```

Comment:
Could we minimize the scope of the variable and initialize it at the point of declaration by removing line 495 and declare the forests variable on line 498?